src/reflection/FStar.Reflection.Basic.fs

```diff
@@ -578,6 +584,12 @@ let pack_sigelt (sv:sigelt_view) : sigelt =
       let se = mk_sigelt <| Sig_bundle (ind_se::ctor_ses, ind_lid::c_lids) in
       { se with sigquals = Noeq::se.sigquals }
 
+    | Sg_Val (nm, us_names, ty) ->
+        let val_lid = Ident.lid_of_path nm Range.dummyRange in
+        let s = SS.univ_var_closing us_names in
+        let typ = SS.subst s ty in
```

Comment:
Similar to above, with` SS.close_univ_vars`.

Reply:
Same as above